src/flint/fq_nmod.jl

```diff
@@ -661,14 +661,14 @@ end
 #
 ###############################################################################
 
-function FlintFiniteField(char::Int, deg::Int, s::AbstractString; cached = true)
+function FlintFiniteField(char::Int, deg::Int, s::Union{AbstractString,Symbol} = :o; cached = true)
```

Comment:
Actually had forgotten to update the constructors in this file. Fixed now, and tests added to verify this.